whipper/common/config.py

```diff
@@ -115,6 +115,22 @@ def getReadOffset(self, vendor, model, release):
             raise KeyError("Could not find read_offset for %s/%s/%s" % (
                 vendor, model, release))
 
+    def getMaxReadSpeed(self, vendor, model, release):
```

Comment:
you don't need to define a custom method here to get the parameter out of the config file: the generic read-cli-opts-from-the-config code will automatically search for `speed` in the config file and use that as a default for the `-s` arg if it's found.